### DISCUSSION TITLE
@cronvel, You mentioned CSS within this project?

### DISCUSSION THREAD

**Discussion by nielsnl68** [cronvel/terminal-kit#170] [Q&A]
Hello @cronvel,

When i got you right you mentioned that you want to use CSS style for coordination system, that is already inside? 
Could you explain why you like that so much, just wondering? 
I find CSS the most user unfriendly system from the late 80st.  

thanks for your time and effort of making this package.

**Comment by cronvel** [comment]
@nielsnl68 CSS is the *de facto* standard, the state of art and the most used/known "language" for styling, and since this project is a Node.js project, a lot of potential users are already exposed to CSS (most Node.js developers have some front-end experiences).

That being said, I have **no** intention of implementing CSS in Terminal Kit: it would be way **overkill** But being aware of this standard while implementing X or Y feature is mandatory, e.g.: re-using well-known keyword or concept or philosophy, at least for everything that work and make sense for a terminal application.

As for your opinion on CSS... to be honest I've done some GTK and QT back when I was still a C++ dev, and I found that awful (especially GTK, which was not even object oriented). I've done a bit of winform too, and I frankly hated it. I'm sure that they all have quite evolved since (I stopped C++ 15 years ago), but few months ago I tried my hand on Gnome extension (which use Javascript with binding to gnome API) and it was still nightmarish. And I'm not going to talk about the crazy way the PDF format works! CSS has its flaws (and I don't like spending too much time coding on the front-end side), but it's still arguably better than anything else I ever touched.

Anyway, since I'm not going to implement CSS, the discussion is a bit pointless ;)

**Reply by nielsnl68** [reply]
I agree, on all points you have made, there are a lot of systems made has there issues indeed. 
I came from the basic->pascal tree and ended up doing much of java(android) javascript(incl. nodejs) lately. 

I'm clad you re not wanted to implement CSS so indeed ... case closed.
